vertx-pojo-mapper-common-test/src/main/java/de/braintags/vertx/jomnigate/testdatastore/TestJsonSerializationObserver.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * #%L
+ * vertx-pojo-mapper-common-test
+ * %%
+ * Copyright (C) 2017 Braintags GmbH
+ * %%
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * #L%
+ */
+package de.braintags.vertx.jomnigate.testdatastore;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.junit.Test;
+
+import de.braintags.vertx.jomnigate.init.DataStoreSettings;
+import de.braintags.vertx.jomnigate.init.ObserverDefinition;
+import de.braintags.vertx.jomnigate.observer.ObserverEventType;
+import de.braintags.vertx.jomnigate.observer.impl.JsonSerializationObserver;
+import de.braintags.vertx.jomnigate.testdatastore.mapper.SimpleMapper;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.Json;
+import io.vertx.ext.unit.TestContext;
+
+/**
+ * Tests to improve correct mapping information for defined observers ( or by settings or annotation )
+ * 
+ * @author Michael Remme
+ * 
+ */
+public class TestJsonSerializationObserver extends AbstractObserverTest {
+  private static final io.vertx.core.logging.Logger LOGGER = io.vertx.core.logging.LoggerFactory
+      .getLogger(TestJsonSerializationObserver.class);
+
+  @Test
+  public void test_Serialization(TestContext context) {
+    Vertx vertx = this.getDataStore(context).getVertx();
+    LOGGER.info("EXISTS: " + vertx.fileSystem()
+        .existsBlocking("/Users/mremme/workspace/vertx/vertx-pojo-mapper/vertx-pojo-mapper-common-test/tmp"));
```

Comment:
correct - actually this test is obsolete, cause it does nothing interesting. removed test